.gitignore

```diff
@@ -27,9 +27,6 @@
 # Ignore uploaded test files
 /public/test
 
-# Ignore deployment packages
-/pkg
-
```

Comment:
not relevant anymore?

Reply:
No, it's part of the deployment mechanism for the UK version